### ISSUE TITLE
Wish: Cash-fusion tx detection for history tab

### ISSUE THREAD

**Issue by zander** [Electron-Cash/Electron-Cash#1942]
Imagine this; I have two machines with the same seed (aka HD wallet) and on one I might have been fusing a bit.
Then on the second machine I start EC and it finds all those transactions that indeed touch my addresses.
But it currently doesn't put any titles there. It doesn't even hide them when in the history you stated you want to hide fusion transactions.

It would be awesome if a new tx in my history could be detected and marked as a fuse tx.

On chat Jonald said: "just look for FUZ lokad prefix in the OP RETURN".

**Comment by cculianu** [comment]
Yes I think it would be possible to do this -- scan for the FUZ OP_RETURN when the tx comes in and put some very similar label to the one in the other wallet.  I actually would like to see this too!
